### PR TITLE
Update test_l1_tvSettings.c

### PR DESCRIPTION
**Pull request by tsenapathy**
hg #34 tvSettings - interface to get number of dimming zones supported

### CODE REVIEW

File: src/test_l2_tvSettings.c [hunk 1]
```diff
@@ -3709,6 +3709,70 @@ void test_l2_tvSettings_RetrieveLDIMShortCircuitStatus(void)
...
+*/
+
+void test_l2_tvSettings_GetNumberOfDimmingZones(void)
+{
+    gTestID = 51;
+    UT_LOG_INFO("In %s [%02d%03d]\n", __FUNCTION__, gTestGroup, gTestID);
+
+    tvError_t ret = tvERROR_NONE;
+    int zoneCount_actual = 0;
+    int zoneCount_expected = UT_KVP_PROFILE_GET_UINT32("tvSettings/LDIMShortCircuitStatus/zone_count");
```
Comment (Anbukannadhasan) [inline src/test_l2_tvSettings.c:3738]: In the yaml, I can see as below

  LDIMShortCircuitStatus:
    enable: true
    size : 160

I am not seeing zone_count in the yaml. Correct me if I am wrong.

**Review by Anbukannadhasan — COMMENTED** [review]
Can you please clarify the yaml information pointed out in L2?

File: src/test_l2_tvSettings.c (hunk 1 above)
Comment (tsenapathy) [reply src/test_l2_tvSettings.c:3738]: Thanks.  Updated the zone_count in yaml.

File: src/test_l1_tvSettings.c [hunk 2]
```diff
@@ -17615,6 +17617,121 @@ void test_l1_tvSettings_negative_GetLdimZoneShortCircuitStatus (void)
...
+ */
+void test_l1_tvSettings_negative_GetNumberOfDimmingZones (void)
+{
+	gTestID = 228;   /* It must be 228 */
+	UT_LOG("In:%s [%02d%03d]", __FUNCTION__,gTestGroup,gTestID);
+
+    tvError_t result = tvERROR_NONE ;
+    unsigned int number_of_dimming_zones = 0;
+
+    if (extendedEnumsSupported == true)
```
Comment (hari22yuva) [inline src/test_l1_tvSettings.c:17705]: extendedEnumsSupported is wrongly implemented in this module 
let's take this has separate issues

**Review by hari22yuva — APPROVED** [review]
changes looks ok

**Review by Anbukannadhasan — APPROVED** [review]
Approved

File: src/test_l1_tvSettings.c (hunk 2 above)
Comment (tsenapathy) [reply src/test_l1_tvSettings.c:17705]: Sorry, couldn't get it.  Could you please explain it ?

**Pull request merged**